deployment/hasura/metadata/databases/tables/sequencing/sequence_adaptation.yaml

```
@@ -0,0 +1,37 @@
+table:
+  name: sequence_adaptation
+  schema: sequencing
+configuration:
+    custom_name: "sequence_adaptation"
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [adaptation]
+      check: {}
```

Minor: missing `set` here to initialize the `updated_by` column:
```suggestion
insert_permissions:
  - role: aerie_admin
    permission:
      columns: [adaptation]
      check: {}
      set:
          updated_by: "x-hasura-user-id"
```

---

I updated and reordered the permissions to be `select, insert, update, delete`